python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
The combo box is too small in XSI.

![loader_status_filter](https://cloud.githubusercontent.com/assets/2866068/8967327/2891a0e8-3604-11e5-90b2-3e82e636d827.png)